physics/eclipse_test.cpp

```diff
@@ -0,0 +1,167 @@
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+#include "physics/solar_system.hpp"
+#include "quantities/quantities.hpp"
+#include "quantities/si.hpp"
+#include "integrators/symplectic_runge_kutta_nyström_integrator.hpp"
+#include "geometry/epoch.hpp"
+#include "geometry/named_quantities.hpp"
+#include "quantities/elementary_functions.hpp"
+#include "geometry/grassmann.hpp"
+#include "testing_utilities/numerics.hpp"
+
+namespace principia {
+
+using astronomy::ICRFJ2000Equator;
+using geometry::JulianDate;
+using integrators::McLachlanAtela1992Order5Optimal;
+using quantities::si::Minute;
+using quantities::si::Metre;
+using quantities::si::Nano;
+using quantities::si::Milli;
+using quantities::si::Kilo;
+using quantities::ArcCos;
```

Comment:
Please move before the `using`s for `quantities::si`.  In general we put less nested declarations (e.g., `quantities::ArcCos`) before more nested ones (e.g., `quantities::si::Anything`).